test/e2e/common/expansion.go

```diff
@@ -157,7 +157,7 @@ var _ = framework.KubeDescribe("Variable Expansion", func() {
 		    Description: Make sure a container's subpath can be set using an
 			expansion of environment variables.
 	*/
-	It("should allow substituting values in a volume subpath [Feature:VolumeSubpathEnvExpansion][NodeAlphaFeature:VolumeSubpathEnvExpansion]", func() {
+	It("should allow substituting values in a volume subpath [Feature:VolumeSubpathEnvExpansion]", func() {
```

Comment:
once a feature is enabled by default, do we drop the `[Feature:...]` tag?

Reply:
that would actually exercise the feature by default in CI

Reply:
Yup, let's drop `[Feature:VolumeSubpathEnvExpansion]` too so that it will run by default